client/my-sites/site-settings/disconnect-site/index.jsx

```diff
@@ -52,6 +149,7 @@ const DisconnectSite = ( { reason, siteSlug, translate } ) => {
 
 const connectComponent = connect( ( state ) => ( {
 	siteSlug: getSelectedSiteSlug( state ),
+	site: getSelectedSite( state ),
```

Comment:
Now when we need more info about the site than just the slug, we can call only `getSelectedSite` and access the slug as `site.slug`.

Reply:
done in 2a7c839ae38cc7a197af66df24bc202f7e3e9a18